src/main/java/core/basesyntax/service/imp/ReportServiceImp.java

```diff
@@ -7,13 +7,18 @@
 
 public class ReportServiceImp implements ReportService {
     private static final String SEPARATOR = ",";
+    private final ProductDao productDao;
+
+    public ReportServiceImp(ProductDao productDao) {
+        this.productDao = productDao;
+    }
 
     @Override
-    public List<String> createReport(ProductDao productDao) {
+    public List<String> createReport(String header) {

```

Comment:
@Override
    public List<String> createReport(String header) {
        List<String> list =  productDao.getAll()
                .entrySet()
                .stream()
                .map(e -> e.getKey() + SEPARATOR + e.getValue().toString())
                .collect(Collectors.toList());
        list.add(0, new String("fruit,quantity"));
        return list;
    }

Reply:
something like this